en/streaming-integrator/docs/setup/deploying-si-as-minimum-ha-cluster.md

```diff
@@ -1,90 +1,136 @@
-      the WSO2 SI nodes in the cluster.
+!!! note
+    - The configurations given below need to be done in the `<SI_HOME>/conf/server/deployment.yaml` file for both
+      the WSO2 SI nodes in the cluster.<br/><br/>
     - If you need to run both SI instances in the same host, make sure that you do a port offset to change the default 
-      ports in one of the hosts. For more information about the default ports, see Configuring Default Ports.
+      ports in one of the hosts. For more information about the default ports, see [Configuring Default Ports](../ref/configuring-default-ports.md).
       
-See below the steps and configuration details need to carry out to configure the HA cluster.
-
-1 . For each node, enter a unique ID for the id property under the wso2.carbon section. (e.g., id: wso2-si). 
-This is used to identify each node within a cluster.
-
-2 . To allow the two nodes to use same persistence storage, you need to configure RDBMS persistence configuration under
- the state.persistence. The following is a configuration for db-based file persistence (Persistent configuration).
- 
-    - state.persistence:
-        enabled: true
-        intervalInMin: 1
-        revisionsToKeep: 2
-        persistenceStore: org.wso2.carbon.stream.processor.core.persistence.DBPersistenceStore
-        config:
-          datasource: PERSISTENCE_DB   # A datasource with this name should be defined in wso2.datasources namespace
-          table: PERSISTENCE_TABLE
+To configure the HA cluster, follow the steps below:
+
+1. For each node, enter a unique ID for the `id` property under the `wso2.carbon` section (e.g., id: wso2-si). This is used to identify each node within a cluster.
+
+
+2. To allow the two nodes to use same persistence storage, you need to configure RDBMS persistence configuration under `state.persistence`. The following is a configuration for db-based file persistence.
+
+    !!! info
+        This step covers persistent configuration.
+
+    ```
+     - state.persistence:
+         enabled: true
+         intervalInMin: 1
+         revisionsToKeep: 2
+         persistenceStore: org.wso2.carbon.stream.processor.core.persistence.DBPersistenceStore
+         config:
+           datasource: PERSISTENCE_DB   # A datasource with this name should be defined in wso2.datasources namespace
+           table: PERSISTENCE_TABLE
+    ```
           
- The datasource named PERSISTENCE_DB in the above configuration can be defined in the 
- <SI_HOME>/conf/server/deployment.yaml file under the wso2.datasources section. Following is a sample 
- datasource configuration.
-    
-    - name: PERSISTENCE_DB
-          description: The MySQL datasource used for persistence
+    The datasource named `PERSISTENCE_DB` in the above configuration can be defined in the `<SI_HOME>/conf/server/deployment.yaml`
+     file under `wso2.datasources`. The following is a sample datasource configuration.
```

Comment:
shall we mention that user can use any db types and mentioned  the ones we support ?

Reply:
Added information about the supported database type in the information note at the beginning of this step.